kannada/romaanchanavee_kannada/romaanchanavee_kannada.md

```diff
@@ -22,3 +22,6 @@
   ೧.  ಹಳಗನ್ನಡ ಕ್ರಿ.ಶ. ೪೫೦ರಿಂದ ಕ್ರಿ.ಶ. ೧೨೦೦ರವರೆಗೆ,
   ೨.  ನಡುಗನ್ನಡ ಕ್ರಿ. ಶ. ೧೨೦೦ರಿಂದ ಕ್ರಿ.ಶ. ೧೭೦೦ರವರೆಗೆ ಮತ್ತು
   ೩.  ಹೊಸಗನ್ನಡ ಕ್ರಿ. ಶ. ೧೭೦೦ರಿಂದ ಪ್ರಸ್ತುತ ಕಾಲಘಟ್ಟದವರೆಗೆ.
+
+ನಿಘಂಟು:
+ಕನ್ನಡದ ಮೊದಲ ಕನ್ನಡ-ಇಂಗ್ಲೀಷ್ ನಿಘಂಟನ್ನು ರಚಿಸಿದವರು ರೆವರೆಂಡ್ ಫರ್ಡಿನಾಂಡ್ ಕಿಟ್ಟೆಲ್. ಈ ನಿಘಂಟು ೭೦,೦೦೦ಕ್ಕೂ ಅಧಿಕ ಕನ್ನಡ ಪದಗಳನ್ನು ಒಳಗೊಂಡಿದೆ. ರೆವರೆಂಡ್ ಫರ್ಡಿನಾಂಡ್ ಕಿಟ್ಟೆಲ್ ಕನ್ನಡ ಭಾಷೆಯ ಮೂರು ಉಪಭಾಷೆಗಳನ್ನೊಳಗೊಂಡ ಪ್ರಧಾನ ವ್ಯಾಕರಣವನ್ನು ವಿವರಿಸುವ ಗ್ರಂಥವೊಂದನ್ನೂ ರಚಿಸಿದ್ದಾರೆ.
```

Comment:
This line is longer than 120 characters. Could you break it up into multiple lines? That way it will be easier to read the raw files or view diffs on the command line.  No need to add blank lines in between, just press enter somewhere in the middle of the line.